locksmith/src/controllers/v2/purchaseController.ts

```diff
@@ -0,0 +1,60 @@
+import { Response, Request } from 'express-serve-static-core'
+import {
+  getStripeCustomerIdForAddress,
+  createStripeCustomer,
+} from '../../operations/stripeOperations'
+import PaymentProcessor from '../../payment/paymentProcessor'
+import * as Normalizer from '../../utils/normalizer'
+import logger from '../../logger'
+const config = require('../../../config/config')
+
+export class PurchaseController {
+  processor = new PaymentProcessor(config.stripeSecret)
+  async createSetupIntent(request: Request, response: Response) {
+    try {
+      const userAddress = Normalizer.ethereumAddress(
```

Comment:
I wonder if we should start doing this in the middleware..

Reply:
hm, maybe. Do we have any case where we do want the normalized address?